src/CommunityToolkit/Xamarin.CommunityToolkit/Extensions/BindableObjectExtension.shared.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System;
```

Comment:
I don't think we need this class. Please drop it.

Reply:
Done